packages/matrix/scripts/register-test-user.ts

```diff
@@ -8,13 +10,36 @@ let password = process.env.PASSWORD || adminPassword;
 
 (async () => {
   return new Promise<string>((resolve, reject) => {
-    childProcess.exec(`docker exec boxel-synapse register_new_matrix_user http://localhost:8008 -c /data/homeserver.yaml -u ${username} -p ${password} --no-admin`,
-      (err, stdout) => {
+    childProcess.exec(
+      `docker exec boxel-synapse register_new_matrix_user http://localhost:8008 -c /data/homeserver.yaml -u ${username} -p ${password} --no-admin`,
+      async (err, stdout) => {
```

Comment:
This needs to be async to support the login afaik, is the use of resolve/reject in here enough to correctly handle this (so I'm not just adding a future weird timing issue)?

Reply:
it makes sense to me!